plugins/README.md

```diff
@@ -0,0 +1,88 @@
+## Relayer Plugins
+
+Relayer plugins are TypeScript functions that can be invoked through the relayer HTTP API.
+
+Under the hood, the relayer will execute the plugin code in a separate process using `ts-node` and communicate with it through a Unix socket.
+
+## Setup
+
+#### 1. Writing your plugin
+
+```typescript
+import { Plugin, runPlugin } from "./lib/plugin";
+
+type Args = {
+  foo: string;
+  bar: number;
+}
+
+async function myPlugin(plugin: Plugin, args: Args) {
+    console.log(args.foo);
+    console.log(args.bar);
+
+    const relayer = plugin.useRelayer("my-relayer");
```

Comment:
Just to understand, plugin configuration is now global, not tied to specific relayer when needed?

Reply:
Yes, correct, that makes it more flexible, you can send transactions through multiple relayers in a single plugin call